tests/tests_ansible.yml

```diff
@@ -333,13 +333,34 @@
           zone: customzone
           state: present
           permanent: yes
+          immediate: yes
         register: result
 
       - name: assert firewalld custom zone
         assert:
           that:
             - result is changed
 
+      - name: Add masquerade to custom zone
+        firewall_lib:
+          masquerade: yes
+          zone: customzone
+          state: enabled
+          permanent: yes
+          immediate: yes
+        register: result
+        failed_when: result is failed or not result.changed
```

Comment:
`result is failed or not result is changed` https://docs.ansible.com/ansible/2.9/user_guide/playbooks_tests.html#task-results